corehq/messaging/scheduling/static/scheduling/js/conditional_alert_main.js

```diff
@@ -34,5 +34,34 @@ hqDefine("scheduling/js/conditional_alert_main", [
         // setup tab
         basicInformation.setRuleTabVisibility();
         $("#conditional-alert-basic-info-panel").koApplyBindings(basicInformation);
+
+        $("#conditional-alert-save-btn").click(function () {
+            var newRuleName = $("#id_conditional-alert-name").val();
+            if (!name) {
+                $("#conditional-alert-form").submit();
+            } else if (name !== newRuleName) {
+                var caseType = $(
+                    "#select2-id_criteria-case_type-container"
+                ).text();
+                var caseCountUrl = $("#conditional-alert-case-count-url").attr(
```

Comment:
There's a pattern for including URLs needed in javascript, check out these docs: https://github.com/dimagi/commcare-hq/blob/master/docs/js-guide/integration-patterns.md#django-urls